middleware/kubernetes/README.md

```diff
@@ -121,6 +121,59 @@ kubernetes coredns.local {
 	# Each line consists of the name of the federation, and the domain.
 	federation myfed foo.example.com
 	
+	# autopath [ndots:NDOTS] [resolv:RESOLV-CONF-FILE] [onnxdomain:RESPONSE] ...
```

Comment:
perhaps ...

autopath [NDOTS [RESPONSE [RESOLV-CONF]]]

I think a custom resolv.conf would be the least likely thing someone would change (out of cluster coredns only), so make that the most inconvenient thing to change.